package cloud.karpov.auth.data

data class GeneralError(var error: Throwable)
